src/ui/public/test_harness/test_sharding/find_bundle_url.js

```diff
@@ -0,0 +1,20 @@
+/**
+ *  We don't have a lot of options for passing arguments to the page that karma
+ *  creates, so we tack some query string params onto the test bundle script url.
+ *
+ *  This function finds that url by looking for a script tag that has
+ *  the "/tests.bundle.js" segment
+ *
+ *  @return {string} url
+ */
+export function findBundleUrl() {
```

Comment:
If we want to hardcode the `/tests.bundle.js` here, we should rename this to `findTestBundleUrl`. If we want to support passing in the string here, we can keep the current name but should pass in `/tests.bundle.js` and  update the comments.